src/markdown-pages/automate-workflows/automated-tagging.mdx

```diff
@@ -0,0 +1,313 @@
+          " ",
+          "That's it! We hope you enjoyed this quick <b>Hello World</b> example. Please give us feedback and ideas in our [Explorer's Hub community](https://discuss.newrelic.com/c/build-on-new-relic).",
+          " ",
+          "<h2>Remember:</h2>Make sure to tear down the infrastructure and app you just provisioned!"
+        ]
+      }]
+    }
+  ],
+
+  "global_tags": {
+    "dxOwningTeam": "DEMO",
+    "dxEnvironment": "development",
+    "dxDepartment": "product",
+    "dxProduct": "Hello"
+  },
+
+  "resources": [
+    {
+      "id": "host1",
+      "display_name": "Hello-Host1",
+      "provider": "aws",
+      "type": "ec2",
+      "size": "t3.micro"
+    }
+  ],
+
+  "instrumentations": {
+    "resources": [
+      {
+        "id": "nr_infra",
+        "resource_ids": ["host1"],
+        "provider": "newrelic",
+        "source_repository": "-b main git@github.com:newrelic/demo-newrelic-instrumentation.git",
+        "deploy_script_path": "deploy/linux/roles",
+        "version": "1.12.1"
+      }
+    ],
+    "services": [
+      {
+        "id": "nr_node_agent",
+        "service_ids": ["app1"],
+        "provider": "newrelic",
+        "source_repository": "-b main git@github.com:newrelic/demo-newrelic-instrumentation.git",
+        "deploy_script_path": "deploy/node/linux/roles",
+        "version": "6.11.0"
+      }
+    ]
+  }
+}
+```
+
+</Step>
+
+<Step>
+
+## Your newly deployed stack!
+When completed successfully, you should see output similar to what's below. Of course, your URLs will be different, but go ahead and open up a browser on the 'app1' url.
+
+```bash
+[INFO] Executing Deployment
+[✔] Parsing and validating Deployment configuration success
+[✔] Provisioner success
+[✔] Installing On-Host instrumentation success
+[✔] Installing Services and instrumentations success
+[INFO] Deployment successful!
+
+Deployed Resources:
+
+  host1 (aws/ec2):
+    ip: 52.90.86.109
+    services: ["app1"]
+    instrumentation:
+       nr_infra: newrelic v1.12.1
+
+
+Installed Services:
+
+  app1:
+    url: http://52.90.86.109:5001
+    instrumentation:
+       nr_node_agent: newrelic v6.11.0
+
+Completed at 2020-08-20 15:11:14 +0000
+```
+
+</Step>
+
+<Step>
+
+## Generate Traffic
+The simple node application that was deployed contains a very simple web page that looks similar to the screen shot below. Follow the instructions on this page to generate traffic that will be sent to New Relic.
+![Hello World Web Site](../../images/automate-helloworld/helloworld-automationdemo.png)
+
+</Step>
+
+<Step>
+
+## Check out these tags!
+Let's head on over to New Relic. We'll use the [search redirect](https://one.newrelic.com/redirect/search/Hello) to quickly find the resources we deployed. (Folks in our European data center should use this [link](https://one.eu.newrelic.com/redirect/search/Hello).)
```

Comment:
There is actually more work to do to support EU users, the user config file needs different URL endpoints for newrelic (collectors...). I have those URLs documented in an internal post, I'm just not sure about referencing them and where.

Reply:
I'll take out the EU endpoints for now.